pyproject.toml

```diff
@@ -118,6 +118,7 @@ module = [
     "collada",
     "collada.source",
     "mpl_toolkits.mplot3d.axes3d",
+    "tqecd",
```

Comment:
In theory we should not have to include `tqecd` here, as it is fully typed.

Reply:
Tests are now passing thanks to changes in `tqecd` (the `py.typed` file was not correctly placed). This should definitely be removed.